apps/teams-test-app/e2e-test-data/store.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Store",
+  "platforms": "Web",
+  "checkIsSupported": {
```

Comment:
If you check this in before 11/6, please add this property:
`"version": ">2.30.0",`

If you check this in on or after 11/6, please add this property:
`"version": ">2.31.0",`

This will make sure that your e2e test is only run against versions of TeamsJS that contain this change.

Reply:
updated